packages/seacas/libraries/ioss/src/exodus/Ioex_BaseDatabaseIO.C

```diff
@@ -771,10 +771,11 @@ namespace Ioex {
             assem->add(ge);
             num_added_entities++;
           }
-          else {
+
+          if (ge == nullptr) {
```

Comment:
I think the overall logic here would be simpler if this `ge == nullptr` test were put right after the `auto *ge = ...` line.  Right now we have a test: 
```
 if (ge != nullptr && !Ioss::Utils::block_is_omitted(ge)) {
```
and then a test: 
```
 if (ge == nullptr) {
```
If the second test was first, then the first test would just be:
```
if (!Ioss::Utils::block_is_omitted(ge))
```

Reply:
That sounds good